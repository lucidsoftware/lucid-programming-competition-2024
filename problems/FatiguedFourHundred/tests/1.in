3
45.00 60.00 75.00 90.00
90.00 75.00 60.00 45.00
99.00 66.00 33.00 66.00