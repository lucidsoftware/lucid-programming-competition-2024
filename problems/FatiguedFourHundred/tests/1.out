75 90 95 100 299.26
90 95 100 100 296.66
90 95 85 100 290.65
